frontend/src/views/Administration.vue

```diff
@@ -189,10 +370,10 @@ export default {
       return this.projectDetails.createdBy
     },
     description () {
-      return this.projectDetails.description
+      return this.projectDetails.description || 'None'
     },
     purpose () {
-      return this.projectDetails.purpose
+      return this.projectDetails.purpose || 'None'
     },
```

Comment:
none should not be shown when editing the field

Reply:
<img width="721" alt="Screenshot 2020-05-08 at 09 56 27" src="https://user-images.githubusercontent.com/5526658/81384607-3736d280-9112-11ea-885b-6c69ed7601d6.png">
I can still press the confirm button even if the value does not match the configured regex

Reply:
![image](https://user-images.githubusercontent.com/1574023/81400646-e8973180-912d-11ea-83de-c02454260233.png)

![image](https://user-images.githubusercontent.com/1574023/81400696-049ad300-912e-11ea-9429-98e0dcf348b2.png)


✔️